tests/backup/backup_test.go

```diff
@@ -8097,6 +8097,24 @@ func GetAllBackupsAdmin() ([]string, error) {
 	return backupNames, err
 }
 
+// GetAllRestoresAdmin returns all the backups that px-central-admin has access to
+func GetAllRestoresAdmin() ([]string, error) {
+	restoreNames := make([]string, 0)
+	backupDriver := Inst().Backup
+	ctx, err := backup.GetAdminCtxFromSecret()
+	log.FailOnError(err, "Fetching ctx")
+
+	restoreEnumerateRequest := &api.RestoreEnumerateRequest{
+		OrgId: orgID,
+	}
+	restoreResponse, err := backupDriver.EnumerateRestore(ctx, restoreEnumerateRequest)
+	log.FailOnError(err, "Failed to enumerate on restore objects")
```

Comment:
Same here, just return error and then handle it with `log.FailOnError()` where you call this func

Reply:
Done. Fixed it.